programs/blockLogTrim/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
eosio project still targets cmake 3.5+, since this cmakelists.txt is part of a larger project lines 1-2 should be removed and line 4 probably should be removed.